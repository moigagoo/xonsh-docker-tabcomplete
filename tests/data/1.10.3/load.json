{
        "arguments": [
                "--help",
                "-i", "--input"
        ],
        "commands": []
}
