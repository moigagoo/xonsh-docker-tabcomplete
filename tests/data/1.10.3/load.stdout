
Usage:	docker load [OPTIONS]

Load an image from a tar archive or STDIN

  --help             Print usage
  -i, --input        Read from a tar archive file, instead of STDIN
